GitUI/UserControls/FileStatusList.cs

```diff
@@ -654,11 +654,15 @@ public void SetDiffs(IReadOnlyList<GitRevision> revisions, Func<ObjectId, GitRev
                 else
                 {
                     // Get the parents for the selected revision
-                    tuples.AddRange(selectedRev.ParentIds?.Select(parentId =>
-                        ((GitRevision, GitRevision, string, IReadOnlyList<GitItemStatus>))(new GitRevision(parentId),
-                            selectedRev,
-                            _diffWithParent.Text + GetDescriptionForRevision(parentId),
-                            Module.GetDiffFilesWithSubmodulesStatus(parentId, selectedRev.ObjectId, selectedRev.FirstParentGuid))));
+                    var multipleParents = AppSettings.ShowDiffForAllParents ? selectedRev.ParentIds.Count : 1;
+                    tuples.AddRange(selectedRev
+                        .ParentIds?
```

Comment:
❗ This will throw, if `ParentIds == null`
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.addrange?view=netframework-4.6.1#exceptions

Reply:
Though line 640 should be:
` var selectedRev = revisions?.FirstOrDefault();`

Reply:
I do not think that revisions can be null on line 640, but there is no harm in adding that.